src/OSPSuite.Core/Domain/ModelCoreSimulation.cs

```diff
@@ -11,7 +11,7 @@ public interface IModelCoreSimulation : IObjectBase, IWithCreationMetaData, IWit
       /// <summary>
       ///    Build configuration used to create the simulation. May be null
       /// </summary>
-      IBuildConfiguration BuildConfiguration { get; }
+      IBuildConfiguration BuildConfiguration { get; set; }
```

Comment:
This may be a breaking change in pksim and Mobi FYI

Reply:
Meaning, we'll have to implement the setters once this is merged?